.ci/Jenkinsfile

```diff
@@ -306,10 +306,14 @@ def runReleaseWithKeychain() {
   }
   def token = getVaultSecret.getVaultToken(vault.vault_addr, vault.vault_role_id, vault.vault_secret_id)
   def apple = getVaultSecret.getVaultSecretObject(vault.vault_addr, 'secret/release/apple', token).data
-  if (!apple.containsKey('keychain-password')) {
-    error("runReleaseWithKeychain: apple secret could not be accessed correctly")
+  if (!apple.containsKey('keychain-password') || !apple.containsKey('username') || !apple.containsKey('app-specific-password')) {
```

Comment:
IIUC, those fields are already available in the existing Vault secret defined in the `L308`, is that correct?

Reply:
Yes that's correct.